styles.css

```diff
@@ -0,0 +1,294 @@
+  background-color: #1a2236;
+  color: white;
+  font-family: 'Poppins', arial, sans-serif;
+  font-size: 1rem;
+  font-weight: 500;
+  line-height: 1.5rem;
+  text-align: center;
+  background-image: url(icons/ic_download.svg);
+  background-repeat: no-repeat;
+  background-position: 75%;
+  margin-top: 1.25rem;
+  margin-bottom: 1.875rem;
+}
+
+.knowledge-container {
+  display: grid;
+  grid-template-rows: 18.063rem 18.063rem 18.063rem;
+  gap: 0.625rem;
+}
+
+.language {
+  background-color: #1c528a;
+  background-image: url(icons/ic_languages_mob.svg);
+  background-repeat: no-repeat;
+  background-position: right bottom;
+  color: white;
+}
+
+.frameworks {
+  background-color: #e2505c;
+  background-image: url(icons/Group\ 18.png);
+  background-repeat: no-repeat;
+  background-position: right bottom;
+  color: white;
+}
+
+.skills {
+  background-color: #febf52;
+  background-image: url(icons/ic_skills.svg);
+  background-repeat: no-repeat;
+  background-position: right bottom;
+  color: white;
+}
+
+.know-h3 {
+  font-family: 'Poppins', arial, sans-serif;
+  font-size: 1.25rem;
+  font-weight: 800;
+  line-height: 1.875rem;
+  margin-left: 4.125rem;
+  margin-top: 3.125rem;
+  margin-bottom: 1.25rem;
+}
+
+.knowledge-container .techlist {
+  font-family: 'Poppins', arial, sans-serif;
+  font-size: 1rem;
+  font-weight: 600;
+  line-height: 1.688rem;
+  margin-left: 4.875rem;
+}
+
+.contact-me {
+  background-color: #f9f9fa;
+  background-image: url(icons/image_geometry_1.png);
+  background-repeat: no-repeat;
+  padding: 6.625rem 0.875rem 1.94rem 1.125rem;
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+}
+
+.getstarted {
+  font-family: 'Playfair Display', arial, sans-serif;
+  font-size: 3.125rem;
+  font-weight: 400;
+  line-height: 3.125rem;
+  margin-bottom: 1.25rem;
+}
+
+.interest {
+  font-family: Poppins, arial, sans-serif;
+  font-size: 0.875rem;
+  font-weight: 400;
+  margin-bottom: 0.875rem;
+}
+
+.contact-me label {
+  font-weight: 400;
+  font-size: 0.625rem;
+  color: #7d7c84;
+  font-family: Poppins, Arial, sans-serif, Helvetica;
+}
+
+.contact-me input {
+  display: block;
+  border-radius: 0.313rem;
+  border: 0.063rem solid lightgray;
+  width: 100%;
+  padding-left: 0.313rem;
+  height: 3.25rem;
+  font-family: Poppins, sans-serif, Arial, Helvetica;
+  font-weight: 400;
+  font-size: 0.875rem;
+}
+
+.contact-me textarea {
+  padding-left: 0.313rem;
+  padding-top: 0.625rem;
+  height: 7rem;
+  display: block;
+  border: 0.063rem solid lightgray;
+  border-radius: 0.313rem;
+  font-size: 0.875rem;
+  font-family: 'Poppins', sans-serif;
+  font-weight: 400;
+  width: 100%;
+}
+
+.footer {
+  background-color: #1a2236;
+  width: 100%;
+  height: 8.563rem;
+}
+
+.social-icons {
+  display: flex;
+  justify-content: space-around;
+  list-style: none;
+  margin-top: 1.25rem;
+}
+
+.get-resume {
+  border: none;
+  background-color: transparent;
+  background-image: url(icons/Union\ \(2\).png);
```

Comment:
In order to fix your stylelint error, add a space before your closing bracket . see below for an example:

`  background-image: url(icons/Union\ \(2\).png );
`